spec/unit/filter_spec.rb

```diff
@@ -133,8 +145,8 @@
       ])
 
       expected = [
-        ElementFilter.new("/unmanaged_files/files/name", ["/home/alfred", "/var/cache"]),
-        ElementFilter.new("/unmanaged_files/files/changes", [["md5", "size"]])
+        ElementFilter.new("/unmanaged_files/files/name", :== => ["/home/alfred", "/var/cache"]),
+        ElementFilter.new("/unmanaged_files/files/changes", :== => [["md5", "size"]])
       ]
```

Comment:
For code calling the filter API:

```ruby
e = ElementFilter.new("/unmanaged_files/files/name")
e.addMatchers("=", ["/home/alfred", "var/cache"])
```

WIthin the implementation of the filter:

```ruby
e = ElementFilter.new("/unmanaged_files/files/name")
e.addMatchers(Filter::OPERATOR_EQUALS, ["/home/alfred", "var/cache"])
```